samples/nrf_rpc/protocols_serialization/server/Kconfig

```diff
@@ -22,6 +22,20 @@ config LOG_BACKEND_RTT
 config LOG_BACKEND_UART
 	default n
 
+if SOC_NRF54L15
+
+config BT_CTLR_ECDH
+	default n
+
+config BT_USE_PSA_API
+	default y if BT_TINYCRYPT_ECC
```

Comment:
What? Why does it require Tinycrypt if it uses PSA API.

Reply:
This is just a wrong Kconfig. BT_TINYCRYPT_ECC only enables BLE host crypto but it uses PSA crypto API if BT_USE_PSA_API is used. I guess Bluetooth team is going to clean this up soon :)